app/classes/Transvision/ShowResults.php

```diff
@@ -145,6 +145,22 @@ public static function getTranslationMemoryResults($strings, $search, $max_resul
     }
 
     /**
+     * Use the API to return a transliterated string
+     *
+     * @param string $string The string to be transliterated
+     * @param string $locale The target locale
+     *
+     * @return string The string transliterated into the target locale
+     */
+    public static function getTransliteratedString($string, $locale)
+    {
+        $request = new API(parse_url(API_ROOT . "transliterate/$locale/$string"));
+        $json = include MODELS . 'api/transliterate.php';
+
+        return isset($json[0]) ? $json[0] : '@@missing@@';
```

Comment:
Call to this method is after checks on `@@missing@@`.  Let's either use an empty string, or something like 'Function not available'.